installer.nsh

```diff
@@ -104,33 +89,34 @@
                         MessageBox MB_YESNO|MB_DEFBUTTON2 "Would you like to unregister your device?" IDYES true IDNO false
                         true:
                             FileWrite $uninstallLog "- ...unregister your device: YES$\r$\n"
-                            StrCpy $ps_command_uninstall 'powershell "& " "$\'"$path_u\remoteit.exe$\'" -j unregister --yes'
-                            nsExec::ExecToStack /OEM $ps_command_uninstall 
+                            
+                            RMDir /r "$APPDATA\remoteit"
+                            FileWrite $uninstallLog "- RMDir $APPDATA\remoteit$\r$\n"
+                            MessageBox MB_OK "Your device was unregistered!" 
+                            Goto next
+                        false:
+                            FileWrite $uninstallLog "- ...unregister your device: NO$\r$\n"
+                            StrCpy $ps_command_uninstall 'powershell "& " "$\'"$path_u\remoteit.exe$\'" -j agent uninstall'
+                            nsExec::ExecToStack /OEM $ps_command_uninstall
                             Pop $0
-                            Pop $1      
-                            FileWrite $uninstallLog "$ps_command_uninstall     [$0]  $1$\r$\n"
-
+                            Pop $1
+                            FileWrite $uninstallLog "-$ps_command_uninstall     [$0]  $1$\r$\n"
```

Comment:
What does this do, and it's duplicated in line 108

Reply:
FileWrite is writing the logs, each action result is in `[$0]` after that we should write the response